components/Resource/resource-table.tsx

```
@@ -31,34 +31,36 @@ export function ResourceTable({
+      <div className="overflow-y-auto max-h-[570px] overflow-x-auto">
+        <table className="w-full min-w-full table-fixed">
+          <thead className="sticky top-0 z-10">
+            <tr className="border-b bg-card text-xs font-medium text-muted-foreground">
+              <th className="text-left p-2.5 w-4/12">Name</th>
+              <th className="text-left p-2.5 w-2/12">Type</th>
+              {showModuleColumn && (
+                <th className="text-left p-2.5 w-2/12">Module</th>
+              )}
+              <th className="text-left p-2.5 w-2/12">Size</th>
+              <th className="text-left p-2.5 w-2/12">Added</th>
+              <th className="text-right p-2.5 w-1/12"></th>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)